src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -114,6 +111,9 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
     addWelcomeMessage()
     ApplicationManager.getApplication().executeOnPooledThread { refreshSubscriptionTab() }
     loadNewChatId()
+
+    // Initiate filling recipes panel in the background
+    refreshRecipes()
```

Comment:
moved from above - probably no actual impact but looks better at the end. I will provide some cleanup to this class in a separate PR